packages/volto/src/config/ControlPanels.js

```diff
@@ -76,7 +76,6 @@ export const unwantedControlPanelsFields = {
     'site_favicon_mimetype',
     'exposeDCMetaTags',
     'enable_sitemap',
-    'robots_txt',
```

Comment:
@sneridagh Should we keep hiding it here if the VOLTO_ROBOTSTXT environment variable is set? Because then robots.txt is served based on the environment variable and not the setting.